mne/forward/_field_interpolation.py

```diff
@@ -193,6 +219,57 @@ def _as_meg_type_evoked(evoked, ch_type='grad', mode='fast'):
     return evoked
 
 
+def _interpolate_bads_meg(inst, mode='fast', verbose=None):
```

Comment:
is `mode='accurate'` appreciably slower? I think we might want it to be the default for channel interpolation, whereas `fast` should be the default for mapping-to-surface (at least from a user API point of view).